config/layouts.php

```diff
@@ -0,0 +1,22 @@
+<?php
```

Comment:
@nathanrice Could I have your help understanding the purpose of this file, please? I can't see that it has any effect. Is it supposed to define default layouts and be a replacement for these lines still present in functions.php?

```php
// Removes site layouts.
genesis_unregister_layout( 'content-sidebar-sidebar' );
genesis_unregister_layout( 'sidebar-content-sidebar' );
genesis_unregister_layout( 'sidebar-sidebar-content' );
```

Reply:
Yes, sorry. I forgot to remove those lines.

I went back and forth on this. Ideally, the child theme should define the layouts it supports, not unregister the ones it doesn't. But I don't see any sense in having to define the layout details twice, so I just pulled the parent config over manually and used them.

One could make the case that Genesis should do merged configs for this, but in that case you're not so much defining the layouts for the child theme, but (again) disabling ones from the parent. So pulling the priority config seemed best to me.

Reply:
@mindctrl @nathanrice You're right — the file has no effect at the moment. We need a way to declare which layouts the theme supports. The options seem to be:

1. Have Genesis auto-detect the layouts.php to set up supported layouts.
2. Use the layouts config file in the functions.php to declare supported layouts (via a filter).
3. Remove layouts.php for now, reinstate the code below in functions.php, and think on (1) and (2) in a future beta release or update.

I think we should consider (3) based on the current release timeline and scope of the new config function (I think it's fine just to showcase it as a way to pass config into `add_theme_support()` for this release).

```php
// Removes unneeded site layouts.
genesis_unregister_layout( 'content-sidebar-sidebar' );
genesis_unregister_layout( 'sidebar-content-sidebar' );
genesis_unregister_layout( 'sidebar-sidebar-content' );
```

Reply:
Got it, thanks. I agree with you on doing (3) for now. I've removed the `config/layouts.php` file and reinstated the code in `functions.php`.